easybuild/tools/github.py

```diff
@@ -1591,6 +1591,25 @@ def update_pr(pr_id, paths, ecs, commit_msg=None):
     print_msg(msg, log=_log)
 
 
+def check_online_status():
+    """
+    Check whether we currently are online
+    Return True if online, else an URLError instance of the last failure
+    """
+    # Try repeatedly and with different URLs to cater for flaky servers
+    # E.g. Github returned "HTTP Error 403: Forbidden" and "HTTP Error 406: Not Acceptable" randomly
+    # Timeout and repeats set to total 1 minute
+    urls = [GITHUB_URL, GITHUB_API_URL]
+    for i in range(6):
```

Comment:
Don't use single-letter variables outside of list comprehensions please. Maybe use `idx` or `iter` or `attempt_id`?

Also clarify the `6`: `total 1 minute (6 * 10s)`?

Reply:
Done: https://github.com/easybuilders/easybuild-framework/pull/3194/commits/87fd425566bbe3827a7ff5ade948dbaadc40a005